### DISCUSSION TITLE
New logic block

### DISCUSSION THREAD

**Discussion by thatisjake** [tsunamayo/Starship-EVO#6053] [Suggestions]
It is a pain when needing to wire up a ton of logic gates after copy-pasting some circuit, It would be real nice to have some block that would emit a signal if any blocks touching it are also on. This would make it so wiring up tons of circuits after copy-pasting a whole bunch isn't necessary, as the output of a circuit would automatically be able to "connect" to the input of another. 
I recently tried to achieve a similar effect with using a piston & laser sensor, but the laser sensor doesn't detect the entity it is apart of, so this didn't work.

**Comment by tsunamayo** [comment]
@thatisjake Hi, well it unfortunately breaks pretty much all the ship builds, it would also be take some resources as it would require a physics query (remember it is not a voxel game, there is no easy way to know what block are touching another one). Cheers
